fairtally/cli.py

```diff
@@ -0,0 +1,76 @@
+import io
+import json
+from pathlib import Path
+import click
+from howfairis import Checker
+from howfairis import Compliance
+from howfairis import Repo
+from jinja2 import Template
+from tqdm import tqdm
+from fairtally.get_badge_color import get_badge_color
+from fairtally.redirect_stdout_stderr import RedirectStdStreams
+
+
+@click.command()
+@click.argument("urls", nargs=-1)
+@click.option("-o", "--output-filename", "output_file", help="Where to write the results of the analysis.",
```

Comment:
Mention output formats in the help text

Reply:
I split this up into a `--html` and a `--json`, each with separate help text; this also avoids having to run the tool twice if you happen to want both json and html output.